packages/vkui/src/components/Touch/Touch.tsx

```diff
@@ -230,15 +230,18 @@ export const Touch = ({
     if (!didSlide.current) {
       return onClickCapture && onClickCapture(e);
     }
-    // eslint-disable-next-line no-restricted-properties
-    if ((e.target as HTMLElement).closest('a')) {
-      e.preventDefault();
-    }
+
     if (noSlideClick) {
       e.stopPropagation();
+
+      // eslint-disable-next-line no-restricted-properties
+      if ((e.target as HTMLElement).closest('a')) {
```

Comment:
```suggestion
      // https://github.com/VKCOM/VKUI/issues/1977
      // https://github.com/VKCOM/VKUI/issues/3892
      if ((e.target as HTMLElement).closest('a')) {
```
давай добавим ссылки на issues, которые чинит это условие

Reply:
Добавил.